NorthstarDLL/squirrel/squirrel.h

```diff
@@ -282,6 +295,51 @@ class SquirrelManagerBase
 		// there are entity constants for other types, but seemingly CBaseEntity's is the only one needed
 		return (T*)__sq_getentityfrominstance(m_pSQVM, &obj, __sq_GetEntityConstant_CBaseEntity());
 	}
+
+	inline SQRESULT createslot(HSquirrelVM* sqvm, const SQChar* key, const SQChar* value, bool isAsset = false, SQInteger idx = -3)
```

Comment:
I just remembered squirrel tables support any type as keys not just tables, if anyone knows a way to make this generic I'd appreciate it

Reply:
I updated `createslot` to be more generic, it should now be possible to insert anything as keys and values, except assets since there isn't a way rn to tell apart sq strings and assets